suites/apis/rasDRSTest.js

```diff
@@ -225,13 +213,25 @@ async function getPassport(I, token) {
 }
```

Comment:
`getTokens` and `getPassport` in this file are no longer used. There are also [some other lint errors](https://app.travis-ci.com/github/uc-cdis/gen3-qa/builds/251768123).